demisto/etc#23400: printing file path instead of file name

hod-alpert opened the pull request:
<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: demisto/etc#23400

## Description
There are lots of README.md files in. content repo, at least one per pack.
When messing with lots of README files it is not enough to know the base name of the file, the full path must be included.

In addition- when there are multiple files with the same names such as README.md it only shows secrets for the last one checked

Itay4 commented:
lets add 2 tests:
1. the actual path is returned and not the name
2. that for more than one readme checked, it returns for them all

hod-alpert commented:
@Itay4 1 is tested in `test_search_potential_secrets__secrets_found` and `test_ignore_entropy`
Added a test for 2
in addition, added fixes to the tests to support legacy pack since it's failing the build

Itay4 commented:
@hod-alpert looks good
lets trigger a content build with secrets in 2 diff READMEs to see it works as expected

hod-alpert commented:
@Itay4 see [here](https://app.circleci.com/pipelines/github/demisto/content/7361/workflows/c5498a89-5aea-4865-a705-d15c2ad49530/jobs/47422)

Itay4 commented:
> @Itay4 see [here](https://app.circleci.com/pipelines/github/demisto/content/7361/workflows/c5498a89-5aea-4865-a705-d15c2ad49530/jobs/47422)

excellent
